eox_hooks/actions_handler.py

```diff
@@ -37,17 +41,32 @@ def action_lookup(module_name, action):
         module = import_module(module_name)
         return getattr(module, action)
     except ImportError:
-        message = "The module {} with the action {} does not exist. A default action will be used."\
-                  .format(
-                      module_name,
-                      action,
-                    )
+        message = "The module {} with the action {} does not exist. A default action will be used.".format(
+            module_name,
+            action,
+        )
     except AttributeError:
-        message = "The action {} does not exist in the module {}. A default action will be used."\
-                  .format(
-                      action,
-                      module.__name__,
-                    )
+        message = "The action {} does not exist in the module {}. A default action will be used.".format(
+            action,
+            module_name,
+        )
 
-    log.warning(message)
+    log.warning("EOX_HOOKS | {}".format(message))
     return custom_action_mock
```

Comment:
Are we still returning a custom_action_mock? I thought this had been removed since we dont want to run anything if there is no action configured. Perhaps that belongs in a different PR

Reply:
I made this PR for that! https://github.com/eduNEXT/eox-hooks/pull/6